### PR TITLE
909 New cloud proxy has SDK version requirement

### PR DESCRIPTION
**Pull request by nirm2009**

### CODE REVIEW

File: shared/variables/global.js [hunk 1]
```diff
@@ -217,6 +219,10 @@ export const IOS_MI_PACKAGE_VERSION = '4.0.0-rc.1';
 export const IOS_PACKAGE = 'NAME: AgoraRtcEngine_iOS';
 export const IOS_VOICE_PACKAGE = 'AgoraRtcEngine_iOS_Voice_Beta';
 
+export const NATIVE = 'Native';
+export const RTC_NATIVE_SDK = `RTC ${NATIVE} SDK`;
+export const RTC_WEB_SDK = `RTC ${WEB} SDK`;
+
```
Comment (atovpeko) [inline shared/variables/global.js:225]: We don't operate with the name RTC SDK though, do we? It's Voice SDK or Video SDK. Also, you probably need to keep the web version for the web page, and the native versions (specifying the platform) for each of the other pages using conditional text.

**Review by atovpeko — CHANGES_REQUESTED** [review]
Please run this by Iain as well

File: shared/variables/global.js (hunk 1 above)
Comment (billy-the-fish) [reply shared/variables/global.js:225]: Video SDK or Voice SDK, certainly not native. Please update any references to this terminology. 

If you must, Video SDK for <Android|iOS|etc.> However, as you are already in a doc that selects by platform, this does not really serve any purpose.

File: shared/video-sdk/develop/_cloud-proxy.mdx [hunk 2]
```diff
@@ -6,11 +6,21 @@ import Reference from '@docs/shared/video-sdk/develop/cloud-proxy/reference/inde
 
 You use <Vg k="COMPANY" /> <Vg k="CP"/> to ensure reliable connectivity for your users when they connect from an environment with a restricted network.
 <PlatformWrapper platform="windows">
-This page is coming soon for Windows,
+This page is coming soon for Windows.
 </PlatformWrapper>
 <PlatformWrapper notAllowed="windows">
 ## Understand the tech of Cloud Proxy
 
+Agora recommends upgrading to the latest SDK version, which support all three operating modes:
```
Comment (atovpeko) [inline shared/video-sdk/develop/_cloud-proxy.mdx:14]:
```suggestion
Agora recommends upgrading to the latest SDK version, which supports all three operating modes:
```

**Review by atovpeko — CHANGES_REQUESTED** [review]

**Comment by billy-the-fish** [comment]
Is this done?

**Pull request merged**